### PR TITLE
Feature/prj model

### PR DESCRIPTION
**Pull request by odeimaiz**

### CODE REVIEW

File: services/web/client/source/class/qxapp/components/form/Auto.js [hunk 1]
```diff
@@ -196,7 +196,7 @@ qx.Class.define("qxapp.components.form.Auto", {
         this.getControl(key).setEnabled(true);
         let upkey = qx.lang.String.firstUp(key);
         let setter = "set" + upkey;
-        let value = data[key];
+        let value = data[key].defaultValue;
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/components/form/Auto.js]: the data object contains the data actual configuation of the node ... there are no defaultValues ...  see the schema of the project ... defaultValues are in the node definition, but they would only be useful when instanciating a new node to fill in default values ... this already happens elsewhere in the code

Comment (oetiker) [reply services/web/client/source/class/qxapp/components/form/Auto.js]: search for defaultValue

Comment (odeimaiz) [reply services/web/client/source/class/qxapp/components/form/Auto.js]: There is a problem when creating new nodes. I implemented a workaround but still fails when the node has a file input. Let's review it tomorrow

File: services/web/client/source/class/qxapp/data/Store.js [hunk 2]
```diff
@@ -9,6 +9,30 @@ qx.Class.define("qxapp.data.Store", {
...
+      return metaData;
+    },
+
+    metaDataToNodeData: function(metaData) {
+      let nodeData = {};
+      nodeData.key = metaData.key;
+      nodeData.version = metaData.version;
+      nodeData.inputs = {};
+      nodeData.outputs = {};
+      for (let inputKey in metaData.inputs) {
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/data/Store.js]: why not write

```
let nodeData = {
   key: metaData.key,
   version: metaData.version
   ...
};
```

**Pull request merged**